test/functional/test_framework/util.py

```diff
@@ -292,6 +292,22 @@ def sha256sum_file(filename):
     return h.digest()
 
 
+def replace_in_config(node, replacements):
```

Comment:
nit: Didn't notice at first, but this is under "# Utility functions", I think it should be under "# Node functions"

Reply:
We should probably remove those comments, given that they are wrong anyway. (`modinv` is under "Block functions", which is wrong)

Reply:
nit: Shouldn't node functions be put under test_node.py, anyway?

Reply:
> nit: Shouldn't node functions be put under test_node.py, anyway?

Good idea, done.